robotframework_reportportal/model.py

```diff
@@ -54,9 +54,14 @@ def __init__(self, name=None, parent_type="SUITE", attributes=None):
             self.status = attributes["status"]
 
     def get_name(self):
-        assignment = "{0} = ".format(", ".join(self.assign)) if self.assign else ""
+        assign = ", ".join(self.assign).encode("utf8")
+        assignment = "{0} = ".format(assign) if self.assign else ""
         arguments = ", ".join(self.args)
-        full_name = "{0}{1} ({2})".format(assignment, self.name, arguments)
+        full_name = "{0}{1} ({2})".format(
```

Comment:
I don't have a problem context and robot framework internal implementation, thus I can be wrong. From my perspective we need to convert all to Unicode first, than build string from parts and then convert single string to utf8 bytes (encode in python3 returns `bytes` type). 
On `python35`
```python
>>> s = "Русский"
>>> print(s)
Русский
>>> type(s)
<class 'str'>
>>> rb=s.encode("utf8")
>>> repr(rb)
"b'\\xd0\\xa0\\xd1\\x83\\xd1\\x81\\xd1\\x81\\xd0\\xba\\xd0\\xb8\\xd0\\xb9'"
>>> repr("{0}".format(rb))
'"b\'\\\\xd0\\\\xa0\\\\xd1\\\\x83\\\\xd1\\\\x81\\\\xd1\\\\x81\\\\xd0\\\\xba\\\\xd0\\\\xb8\\\\xd0\\\\xb9\'"'
```
I am not sure this is desired behavior.